### ISSUE TITLE
onBlur() validation message not showing up

### ISSUE THREAD

**Issue by millturnK** [dockwa/simple-react-validator#93]
Hi guys,
Thanks for implementing onBlur after installing version 1.1.0. I have trialled in one of my input boxes. When onBlur is triggered, your showMessageFor function is being called (why do we pass null into it?) with all the right inputs and the validation rules are being applied but no message is showing up in the browser. Below is the code (excerpts). Let me know if you need any further info.
Many thanks,
Katie

```jsx
constructor(props) {
  super(props);
  this.handleQuestionChange = this.handleQuestionChange.bind(this);
  this.handleSubmit = this.handleSubmit.bind(this);
  this.validator = new SimpleReactValidator({
    autoForceUpdate: this,
    messages: {
      default: 'Validation has failed!',  // will override all messages
      meter: 'Invalid meter number: must be a 10 or 11 digit number'
    }
  });
}

<div className="col-md-4">
  <label>Meter Number</label>
  <input
    type="text"
    id="meterNumInput"
    name="meterNumInput"
    placeholder="enter a 10 or 11 digit meter number"
    onChange={this.handleMeterNumChange}
    onBlur={this.validator.showMessageFor.bind(null, 'meter')}
    className="form-control"
    value={this.state.meterNum}
  />
  <div className="small">
    {this.validator.message('meter number', this.state.meterNum, 'required|min:1000000000,num|max:99999999999,num')}
  </div>
</div>
```

**Comment by stuyam** [comment]
The `.bind(null,` part does nothing, it is just a way to pass in `meter` and have it be deferred to only when the onBlur is called. This will also work:
```jsx
onBlur={() => this.validator.showMessageFor('meter')}
```
Maybe that is more clear?

As for it not working for you. Does it work when you hit submit and turn on all the messages, do they still show up? I tried you example and it was working for me. I need to put together a codepen template that people can use to show examples because it would be much easier if we both had the same broken example to work off of. Are you able to make it into a codepen to try to reproduce?

**Comment by stuyam** [comment]
I updated the docs to use the arrow function instead of the `bind(null` to simplify it, thanks for the feedback on that: https://github.com/dockwa/simple-react-validator#onblur

**Comment by millturnK** [comment]
Beauty guys! The arrow function fixed it (I assume!). Thanks so much - very fast work :-). I'll let you close the issue.

**Comment by stuyam** [comment]
Glad to hear it is working for you! I'm happy knowing the feature was immediately useful for people. 🎉
